### ISSUE TITLE
Rollup changes are not reflected during watch mode. [Linux]

### ISSUE THREAD

**Issue by kiuKisas** [Elderjs/elderjs#168]
So, I don't know since when that happen as I used to often reload rollup due to #141 
Rollup only reload a file the first time it change. After that, my terminal regarding rollup doesn't change.  Elderjs reload as expected though (but doesn't apply the change).

**Comment by timeshift92** [comment]
@nickreese I sent you an invitation there client and server

**Comment by timeshift92** [comment]
@nickreese  i will find your in discord but nout fainded, my discord Нурбек Ахмедов#7361

**Comment by nickreese** [comment]
As an update to this. @kiuKisas Have compared notes and this appears to be a linux bug.

Essentially rollup is not outputting recompiled files into the `./___ELDER___/components` folder.

**Comment by nickreese** [comment]
This appears to be a rollup issue with linux. 

"Unix/Linux uses `inotify`, OSX uses `FSWatch`"[1](https://stackoverflow.com/questions/13695046/watch-a-folder-for-changes-using-node-js-and-print-file-paths-when-they-are-cha/13705878#13705878) to implement file watching. 

According to @lukeed we can probably just override `watch.chokidar:true` in our rollup process. @kiuKisas is testing it.

**Comment by lukeed** [comment]
Sorry, it used to be `true` in an older Rollup version. Now it's the full Chokidar options object: https://rollupjs.org/guide/en/#watchchokidar

Just a guess, but maybe something like this?

```js
// rollup.config.js
watch: {
  chokidar: {
    usePolling: !/^(win32|darwin)$/.test(process.platform)
  }
}
```

**Comment by nickreese** [comment]
Fixed in 1.4.0
